### PR TITLE
Add client ada accommodation backend (DEV-714)

### PR DESCRIPTION
**Pull request by mikefeldberg**
DEV-714

### CODE REVIEW

**Comment by Davit-BetterAngels** [comment]
@mikefeldberg maybe missing but can't see adaAccomodationOther field

**Review by Davit-BetterAngels — COMMENTED** [review]
where in the PRD I need to see like of accommodation OTHER needs to have accommodationOther or not ?

File: apps/betterangels-backend/clients/enums.py [hunk 1]
```diff
@@ -3,6 +3,14 @@
 from django.utils.translation import gettext_lazy as _
```
Comment (Davit-BetterAngels) [inline apps/betterangels-backend/clients/enums.py:3]: where in the PRD I need to see like of accommodation OTHER needs to have accommodationOther or not ?

Comment (mikefeldberg) [reply apps/betterangels-backend/clients/enums.py:3]: it's in the ticket https://betterangels.atlassian.net/browse/DEV-714

> “other” does not include a custom text input

**Review by Davit-BetterAngels — APPROVED** [review]

**Pull request merged**